odoo/addons/test_lint/tests/test_checkers.py

```diff
@@ -458,3 +458,76 @@ def wrapper2(var):
         """)
         with self.assertMessages("sql-injection"):
             checker.visit_call(list(node.get_children())[1])
+
+
+@unittest.skipUnless(pylint and pylint_bin, "testing lints requires pylint")
+class TestI18nChecks(TransactionCase):
+
+    def check(self, test_content):
+        with tempfile.NamedTemporaryFile(mode="w", encoding="utf-8", delete=False) as f:
+            self.addCleanup(os.remove, f.name)
+            f.write(dedent(test_content).strip())
+        res = run(
+            [
+                pylint_bin,
+                f"--rcfile={os.devnull}",
+                "--load-plugins=_odoo_checker_gettext",
+                "--disable=all",
+                "--enable=gettext-variable,gettext-placeholders,gettext-repr",
+                "--output-format=json",
+                f.name,
+            ],
+            stdout=PIPE,
+            encoding="utf-8",
+            env={
+                **os.environ,
+                "PYTHONPATH": HERE + os.pathsep + os.environ.get("PYTHONPATH", ""),
+            },
+            check=False,
+        )
+        return res.returncode, json.loads(res.stdout)
```

Comment:
Assuming this is the exact same code as in `TestSqlLint`, I would create an abstract class with this method that both this and `TestSqlLint` inherit from